include/Game/Entities/Kabuto.h

```diff
@@ -14,6 +163,138 @@ struct ProperAnimator : public EnemyAnimatorBase {
 	SysShape::Animator m_animator; // _10
 };
 } // namespace Kabuto
+
+////////// Armored Cannon Beetle Larva
```

Comment:
no need for the massive comment lol, will be annoying later on because you have to count the comment count
just a little // or even a documentation complicit /* name */ will suffice

Reply:
agreed - adjusted accordingly